### PR TITLE
Add MicroProfile and yaml words

### PR DESCRIPTION
**Pull request by angelozerr**
Add MicroProfile and yaml words

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,25 +20,43 @@ The following commands are supported for both Maven and Gradle Quarkus projects:
   * `Quarkus: Add extensions to current project`: Add Quarkus extensions to currently opened Quarkus project
   * `Quarkus: Debug current Quarkus project`: Launches the Maven `quarkus:dev` plugin or the Gradle `quarkusDev` command and automatically attaches a debugger  
 
-## Quarkus `application.properties` Features
-  * Completion support for Quarkus properties
-  * Hover support for Quarkus properties
-  * Validation support for Quarkus properties 
-  * Support for Quarkus profiles
-  * Outline support (flat or tree view)
+## Quarkus / MicroProfile `properties` Features
```
Comment (xorye) [inline README.md]: I think in general it is better to remove spaces before and after forward slashes:
https://www.grammarly.com/blog/slash/
https://english.stackexchange.com/questions/2131/should-i-write-module-theme-or-module-theme

File: README.md [hunk 2]
```diff
@@ -20,25 +20,43 @@ The following commands are supported for both Maven and Gradle Quarkus projects:
...
-This extension provides several code snippets, available when editing Java files:
+In `application.properties` and `microprofile-config.properties` files, you will benefit with:
 
-  * **qrc** - Create a new Quarkus resource class
-  * **qrm** - Create a new Quarkus resource method
-  * **qtrc** - Create a new Quarkus test resource class
-  * **qntrc** - Create a new Quarkus native test resource class
+  * Completion support for Quarkus / MicroProfile properties
+  * Hover support for Quarkus / MicroProfile properties
+  * Validation and Quickfixes support for Quarkus / MicroProfile properties
```
Comment (xorye) [inline README.md]: Quick Fix support

File: README.md [hunk 3]
```diff
@@ -20,25 +20,43 @@ The following commands are supported for both Maven and Gradle Quarkus projects:
...
+  * Support for Quarkus profiles
+  * Outline support (flat or tree view)
+  * Code snippets:
+    * **qds** - Configure a Quarkus datasource
+    * **qj** - Configure a Jaeger tracer
+  
+## Quarkus `yaml` Features
 
-When editing `application.properties` files, you have access to:
+In `application.yaml`, `application.yml` files, you will benefit with:
```
Comment (xorye) [inline README.md]: In `application.yaml` and `application.yml` files

File: README.md (hunk 1 above)
Comment (xorye) [reply README.md]: ie, Quarkus/MicroProfile instead of Quarkus / MicroProfile

File: README.md (hunk 2 above)
Comment (angelozerr) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (angelozerr) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (angelozerr) [reply README.md]: fixed

Comment (angelozerr) [reply README.md]: Thanks @xorye for the links

**Pull request merged**